barretenberg/cpp/src/barretenberg/plonk_honk_shared/composer/composer_lib.hpp

```diff
@@ -54,7 +54,7 @@ void construct_lookup_read_counts(typename Flavor::Polynomial& read_counts,
 {
     const size_t tables_size = circuit.get_tables_size();
     // TODO(https://github.com/AztecProtocol/barretenberg/issues/1033): construct tables and counts at top of trace
-    size_t table_offset = dyadic_circuit_size - tables_size;
+    size_t table_offset = dyadic_circuit_size - tables_size - MASKING_OFFSET;
```

Comment:
it's like that someone modifying the `table_offset` is not going to be exactly aware of this additional MASKING_OFFSET and all the places it has to be modified so I wonder if subtractions can be unified a bit.